caps/views.py

```diff
@@ -97,6 +103,19 @@ def get_context_data(self, **kwargs):
         except DataPoint.DoesNotExist:
             context["no_emissions_data"] = True
 
+        context["current_emissions_breakdown"] = council.current_emissions_breakdown()
```

Comment:
think this bit needs to be be wrapped in a check for `no_emission_data` set above to get the test workin.